packages/babel-preset-app/README.md

```diff
@@ -98,12 +98,12 @@ yarn add --dev core-js@2 @babel/runtime-corejs2
 export default {
   build: {
     babel: {
-      presets({ isServer }) {
+      // envName: server, client, modern
+      presets({ envName }) {
         return [
           [
-            require.resolve('@nuxt/babel-preset-app'),
+            '@nuxt/babel-preset-app',
```

Comment:
Not sure why it was required as before. I hope it is not breaking NPX.

Reply:
The difference is the resolving happens in `node_modules/@babel/..` instead of `nuxt.config.js`, I think they should have same resolving result in NPM, how do you think ?

Reply:
Me too. As long as explicit dependencies are there (and webpack has explicit dependenc on babel-preset-app) so at least _seems_ to be safe :)